docs/development/testguide.rst

```diff
@@ -45,7 +45,7 @@ separately.
 setup.py test
 =============
 
-Astropy and the affiliated package template provide a ``test`` setup command,
+astropy and the Astropy package template provide a ``test`` setup command,
```

Comment:
The first instance here should be 'The astropy core package and ...' to avoid the missing capitalization